14/umbraco-commerce/release-notes/v14.0.0.md

```diff
@@ -0,0 +1,126 @@
+
+## Key Takeaways
+
+* [New backoffice UI](#new-backoffice-ui) build with web-components.
+* Some [breaking changes](#breaking-changes) to be expected.
+* Brand new [Management API](#management-api).
+* New more flexible [UI extension points](#ui-extension-points) added.
+* New [Management API](#management-api) added.
+* [Localization](#localization) support added (in progress).
+
+## New Backoffice UI
+
+As with everything v14, this is a complete rebuild of the backoffice UI using web-components. This ensures Umbraco Commerce is set for the future and creates a solid foundtion to develop new features and functionality.
+
+![Umbraco Commerce v14 Order Editor](../media/v14/order-editor.png)
+
+![Umbraco Commerce v14 Settings Section](../media/v14/store-settings.png)
+
+## Breaking Changes
+
+With the new backoffice UI, there has inevitably been the need to change some functionalities and use such breaking changes couldn't be avoided. These have tried to be minimized as much as possible, whilst also ensuring we embrace the new front end architecture to it's fullest.
+
+The key breaking changes to expect are:
+
+* **UI Config Files Removed**
+
+  In previous versions of Umbraco Commerce it was possible to configure the back office order list view, order editor and analytics dashboards using JSON UI config files. These have now all been updated to use the new manifests pattern in Umbraco v14. See the [UI Extension Points](#ui-extension-points) section bellow for more details.
+
+  In addition to this, the UI Config Files were also used in the backoffice to extract key order properties for things like extracting the shipping address in order to calculate shipping rates. This functionality has now been split such that this mapping functionality has now moved to a new server based configuration API.
+
+  ```
+  builder.WithOrderPropertyConfigs()
+    .Add("myStoreAlias", b => b
+        .For(x => x.Customer.FirstName).MapFrom("firstName")
+        .For(x => x.Customer.LastName).MapFrom("lastName")
+        ...
+    );
+  ```
+
+* **Web Notification Events Removed**
+
+  The following notifications are no longer being fired.
+
+  * `AnalyticsDashboardConfigParsingNotification`
+  * `CartEditorConfigParsingNotification`
+  * `OrderEditorConfigParsingNotification`
+  * `StoreActionsRenderingNotification`
+  * `ActivityLogEntriesRenderingNotification`
+
+* **Web Controllers / Models Removed**
+
+  With the new Management API this does mean any controllers / models that previously lived in the `Umbraco.Commerce.Cms.Web` will now have been removed. Please use the Management API instead.
+
+  The only exception to this is the `PaymentController` which is used as a callback for payment gateways. This is currently kept at the same URL to prevent breakages for any existing transaction communications, however all new integrations should communicate with a new Payments API.
```

Comment:
```suggestion
The only exception to this rule is the `PaymentController`, which serves as a callback for payment gateways. It is currently located at the same URL to avoid disrupting existing transaction communications. However, all new integrations should communicate with a new Payments API.
```